content/ux-strat-bootcamp/part-one/task-five/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Task Five
+content_type: topic
+prerequisites:
+  hard:
+  - ux-strat-bootcamp/part-one/task-four/data-dashboard-analysis
+ready: true
+---
+
+# **Dashboard Analysis and Visualisation** 
+
+{{% contentlink path="ux-strat-bootcamp/part-one/task-five/dashboard-analysis-and-visualisation" %}}

```

Comment:
This file probably shouldn't exist since it is a project pretending to be a topic. If this is a card on someone's board, then that bootcamper will be able to justy move it to the "complete" column by themselvers without any review. 

Rather put a link to the actual project on the board. If you put things in the right order and set up the prerequisites properly then that would make the syllabus just work.

Topics cards: can be moved around at will by the student (unless the card is blocked by an unmet prerequisite)
Project cards require review and submissions

If you just link to a project from the middle of a topic then Tilde wont know ta\hat you actually want folks to do the work. To tell Tilde that the assignment should be completed it needs to be added as a prerequisaite or as a link on the suyllabus page.